app/main/forms/brief_forms.py

```diff
@@ -0,0 +1,91 @@
+from flask_wtf import Form
+from wtforms import IntegerField, SelectMultipleField
+from wtforms.validators import NumberRange
+
+
+class BriefSearchForm(Form):
```

Comment:
> I'm just talking about the search/filtering forms, which from what I could tell are actually implemented in a fairly separate way.

This is kind of my point, I'd like to have fewer separate implementations for similar things.  Building up an HTML form, validating, and joining parameters based on question type (checkboxes or radio buttons) feel like enough similarities to justify a shared implementation.

> At least this being here would be a strawman example of how such a form could operate.

We've done the different approach before with frontend validation for supplier declarations and we still haven't managed to make it fit with the rest of the code.

Anyway, I'm 10000 miles away, so I'll leave the decision to you and the team.